opampserver/pkg/sdkconfig/instrumentationconfigs_controller.go

```diff
@@ -33,6 +33,11 @@ func (i *InstrumentationConfigReconciler) Reconcile(ctx context.Context, req ctr
 		}
 	}
 
+	if instrumentationConfig.Spec.RuntimeDetailsInvalidated {
+		// This reconciler is only interested in InstrumentationConfig objects that have their RuntimeDetailsInvalidated field set to false
+		return ctrl.Result{}, nil
+	}
```

Comment:
can't this lead to race conditions with us potentially ignoring important updates?